backend/database/migrations/2022_03_04_081257_create_words_table.php

```diff
@@ -16,6 +16,7 @@ public function up()
         Schema::create('words', function (Blueprint $table) {
             $table->id();
             $table->foreignId('category_id');
+            $table->string('word');
```

Comment:
create new migration file for adding new column.

This will only work on `php artisan migrate:fresh` and it is not advisable when working in teams.